sdk/firstload/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "highlight.run",
-	"version": "5.4.2",
+	"version": "5.5.0",
```

Comment:
Shouldn't we bump this to `6.0.0` since it's sort of a breaking change?

Reply:
Updated rrweb and bumped the major version.